generators/add-action/customer-profile/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class CustomerProfileGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {
+      description: 'This is a sample action showcasing how to access an external Customer Profile API',
+      // eslint-disable-next-line quotes
+      requiredParams: `['tenantId', 'iMSOrgId', 'apiKey']`,
+      // eslint-disable-next-line quotes
+      importCode: `const CustomerProfileSDK = require('@adobe/aio-lib-customer-profile')`,
+      responseCode: `// initialize sdk
+    const client = await CustomerProfileSDK.init(params.tenantId, params.iMSOrgId, params.apiKey, token)
+    // call methods, eg getSegmentRoute
+    const response = await client.getExperienceEvents({
+      'schema.name': '_xdm.context.experienceevent',
+      'relatedSchema.name': '_xdm.context.profile',
+      entityIdNS: 'email',
+      entityId: params.email,
+      fields: params.fields,
+      orderby: /^[+-]timestamp$/.test(params.orderby) ? params.orderby : '-timestamp'
+    })`
+    }
+  }
+
+  async prompting () {
+    this.props.actionName = await this.promptForActionName('interacts with the Customer Profile API', 'customer-profile')
```

Comment:
Could you rename the prompt content from `Customer Profile API` into `Adobe Experience Platform: Realtime Customer Profile`?

Reply:
Updated